clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -318,6 +319,10 @@ public enum Command {
       new Arg[] { FABRIC, SERVER_KAFKA_FETCH_QUOTA_RECORDS_PER_SECOND, ALLOW_STORE_MIGRATION,
           CHILD_CONTROLLER_ADMIN_TOPIC_CONSUMPTION_ENABLED }
   ),
+  UPDATE_DARK_CLUSTER_CONFIG(
+      "update-dark-cluster-config", "Update dark cluster configs", new Arg[] { URL, CLUSTER },
```

Comment:
Could you please add more info in this helper doc? For example, if nothing is specified in the store replicate list, what will happen?

Reply:
Added more details. The details about the args can be found at the arg definition.